### DISCUSSION TITLE
Sparse infill direction vs solid infill direction

### DISCUSSION THREAD

**Discussion by simonvez** [OrcaSlicer/OrcaSlicer#5214] [Ideas]
It would be terribly useful to have control of the orientation. right now you can only specify one orientation that will apply to BOTH sparse infill and solid infill. Splitting it in 2 different settings would be awesome

**Comment by mtc1122** [comment]
This seems to be a fairly common request. I've seen it on Reddit as well as in various posts and bug reports here. However, for now, you can use the height range modifiers to achieve specific directions. Of course this take more time. And, the current implementation of adding height range modifiers is not exactly intuitive or user friendly.

**Comment by SoftFever** [comment]
It's been on my to-do list for a while, and I plan to do more than just support setting a solid infill angle separately - I'll also allow users to specify how it rotates.

**Comment by SoftFever** [comment]
Hi
This feature has now been added into the nightly build: https://github.com/SoftFever/OrcaSlicer/releases/tag/nightly-builds

https://github.com/SoftFever/OrcaSlicer/assets/103989404/9ccdb6e7-2030-4e73-a2a2-0691102181a4

**Reply by hsilvey5** [reply]
That's a really neat feature to have, do you think adding in variable infill directions is possible in OrcaSlicer? I'm used to using Simplify v4 which allowed you to input a string of angles it would parse through layer to layer, allowing me to create a more variable lattice structure for my parts. Currently I have 16 modifier blocks the height of a layer stacked all the way to the top of the bed to create the same output in Orcaslicer since im trying to slice to my new bambu (I used to slice for creator pros with simplify v4). Basically 20 degrees layer 1, 120 layer 2, and so on until I've gone through all increments of 10 and then it recycles.

**Comment by simonvez** [comment]
awesome work man !!! thanks a bunch
